extensions/2.0/Khronos/KHR_avatar_skeleton_bindpose/README.md

```diff
@@ -0,0 +1,119 @@
+# KHR_avatar_skeleton_bindpose
+
+## Contributors
+
+- Ken Jakubzak, Meta
+- Hideaki Eguchi / VirtualCast, Inc.
+- K. S. Ernest (iFire) Lee, Individual Contributor / https://github.com/fire
+- Shinnosuke Iwaki / VirtualCast, Inc.
+- 0b5vr / pixiv Inc.
+- Leonard Daly, Individual Contributor
+- Nick Burkard, Meta
+
+## Status
+
+**Draft** – This extension is not yet ratified by the Khronos Group and is subject to change.
+
+## Dependencies
+
+Written against the glTF 2.0 specification.  
+Dependent on: `KHR_avatar`
+
+## Overview
+
+**THIS EXTENSION IS VERY MUCH WORK IN PROGRESS**
+
+The `KHR_avatar_skeleton_bindpose` extension defines a canonical bind pose for the avatar's skeleton. This pose serves as a neutral reference that supports animation retargeting, mesh deformation validation, and consistency across toolchains. It is particularly useful when the glTF `skins[].inverseBindMatrices` are insufficient or absent for expressing the true skeletal rest position.
```

Comment:
We used to define the avatar's T-pose using its default node hierarchy in VRM, and the stance is the single source of truth.

I'm not understanding when we need this information. Is it for avatars that use A-pose for the bindpose and using the T-pose for its default hierarchy, and it might provide information that can be useful for retarget algorithms? In that case, is it optional to have?

Reply:
We could enforce T-pose as we do currently and then provide A-Pose or natural pose as KHR_avatar_expression_procedural as it is theoretically possible.

Another contributor mentioned much like the "jump" example an A-pose or a natural pose would be a creative use as the design intention is for KHR_avatar_expression_procedural to be for things like joint joy, morph shape lip raise and switching the face texture from neutral to `^_^`.

Reply:
I'm not sure I like the idea of overloading the expression_procedural for A-Pose (though natural/idle pose makes sense). 

Procedurals certainly attribute what *can* be driven procedurally; but it doesn't mean that's the only source that could define how those expressions are used. We have to still consider how emotes/full animations will play with these extensions down the line. 

That being said; us having a recommendation as to whether creators should use T-Pose or A-Pose is definitely not out of question; I just think that we need to acknowledge that certain runtimes/pipelines will have opinions that we might need to respect/enable.